### PR TITLE
Centos7

### PR DESCRIPTION
**Pull request by sebastianwebber**
Hi There,

I make a few modification on your module, trying too improve user permissions and swap min size for the extra file created by the swap.sh script.

I test this modifications on a centos7 box on vagrant (box name is: "puppetlabs/centos-7.0-64-puppet").

As bonus, a create a profile script to export de ORACLE_ variables (ORACLE_HOME, ORACLE_SID) and adjust the PATH.

Thanks a lot for this module!

Regards,

### CODE REVIEW

File: files/swap.sh [hunk 1]
```diff
@@ -28,13 +29,19 @@ then
...
+  SWAP_TO_CREATE=$MKSWAP_MIN
+fi
+
+
 if [ "$SWAP" == "none" ]
 then
   exit 0
 else
-  TMP=`mktemp`
+  TMP='/var/oracle-xe.swapfile'
```
Comment (bpholt) [inline files/swap.sh:41]: Why did you change this to a fixed name instead of the dynamic name returned by `mktemp`?

Comment (sebawebber) [inline files/swap.sh:41]: My bad, i forgot that.

I change that for two reasons:
- /tmp files are erased on reboot[1] on Centos 7
- the better way to solve that problem its creating a file out it and add one line at the `/etc/fstab` file, something like this:

```
/var/oracle-xe.swapfile   swap    swap    sw  0   0
```

I found a easy way to extend swap on centos 7, take a look[2].

[1] http://wiki.centos.org/TipsAndTricks/TmpOnTmpfs
[2] https://www.digitalocean.com/community/tutorials/how-to-add-swap-on-centos-7
